eslint.config.mjs

```
@@ -0,0 +1,59 @@
+  {
+    languageOptions: { 
+      globals: globals.browser 
+    }
+  },
+  pluginJs.configs.recommended,
+  ...tseslint.configs.recommended,
+  pluginReact.configs.flat.recommended,
+  {
+    rules: {
+      // React rules
+      'react/react-in-jsx-scope': 'off', // Not needed in React 17+
```

Biome (1.9.4)

[error] This property value named react/react-in-jsx-scope is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named react/react-in-jsx-scope


(lint/suspicious/noDuplicateObjectKeys)